src/runtime/client-event-manager.js

```diff
@@ -91,37 +98,21 @@ export class ClientEventManager {
     this.filterers_.push(filterer);
   }
 
-
   /**
    * @overrides
```

Comment:
nit: Please add a comment describing the type of the "event" argument.

Reply:
I'd previously been told by Dima to take that comment out because it's documented in the API and that's where the documentation should be, rather in than in the override which just matches the API